src/components/forms/NetworkDevicesForm/NetworkDeviceIPAddress.tsx

```diff
@@ -0,0 +1,75 @@
+import { type FC } from "react";
+import type { InstanceAndProfileFormikProps } from "../instanceAndProfileFormValues";
+import { Input } from "@canonical/react-components";
+import { type LxdNetwork } from "types/network";
+import { type LxdNicDevice } from "types/device";
+import { typesWithNicStaticIPSupport } from "util/networks";
+
+interface Props {
+  formik: InstanceAndProfileFormikProps;
+  index: number;
+  network?: LxdNetwork;
+  device: LxdNicDevice;
+  readOnly: boolean;
+  family: "IPv4" | "IPv6";
+}
+
+export const NetworkDeviceIPAddress: FC<Props> = ({
+  formik,
+  index,
+  network,
+  device,
+  readOnly,
+  family,
+}) => {
+  if (!network || !network.config) {
+    return null;
+  }
+
+  const addressString = family === "IPv4" ? "ipv4.address" : "ipv6.address";
+  const networkIP = network.config[addressString];
+  const deviceIP = device[addressString];
+  const isStaticIPAllowed =
+    network.managed && typesWithNicStaticIPSupport.includes(network.type);
+  const canSelectIP = isStaticIPAllowed && networkIP && networkIP != "none";
+
+  if (readOnly) {
+    return (
+      <div className="ip-content">
+        <>
+          <div>{family}</div>
+          <p className="u-no-margin--top u-no-margin--bottom">
+            {deviceIP || "Dynamic"}
+          </p>
```

Comment:
```suggestion
          <div class="mono-font">
            <b>{deviceIP || "Dynamic"}</b>
          </div>
```
If we don't need the spacing of a paragraph, better use a `div` here. Seems also to be the better choice semantically. 

Also we should use the mono font in bold to clarify this is the value.